init.sql

```diff
@@ -0,0 +1,8 @@
+GRANT ALL PRIVILEGES ON app_db.* TO 'app_user'@'%';
```

Comment:
Granting ALL PRIVILEGES to 'app_user' can be risky as it gives the user full control over the database. Consider limiting the privileges to only what is necessary for the application's functionality.